c2corg_ui/static/partials/feed.html

```diff
@@ -26,7 +26,7 @@ <h4 class="text-success text-center" translate ng-show="feedCtrl.busy">Loading f
 
     <p class="action-line">
       <span class="action" ng-cloak>
-        <b><a ng-href="todo">{{doc['user']['name']}}</a></b>
+        <b><a ng-href="/profiles/{{doc['user']['document_id']}}/{{doc['user']['locales'][0]['lang']}}">{{doc['user']['name']}}</a></b>
```

Comment:
> {{doc['user']['locales'][0]['lang']}}

I opened a ticket on the API (https://github.com/c2corg/v6_api/issues/423) because I thought that you would not need the locales of a user profile when getting a user in the listing version. But the way you are doing it here, means that you actually need the locale, right?